src/locales/en.json

```diff
@@ -4,67 +4,67 @@
   "calculator": {
     "precautions": {
       "label": "Precautions",
       "environment_header": "Environment",
       "environment_question": "Is it indoor or outdoor?",
-      "no_intimate_bonus_outdoors": "Due to very close distances, we are not confident that being outdoors reduces the risk in a substantial way. Thus, we are not providing any bonus for being outdoors when intimate.",
+      "no_intimate_bonus_outdoors": "Due to very close distances, being outdoors probably doesn't reduce the risk substantially. There is no bonus for being outdoors when intimate.",
       "your_mask_header": "Your mask",
       "your_mask_question": "What mask are YOU wearing?",
       "your_mask_note": "If you’re eating or drinking, select “no mask”",
       "their_mask_header": "Their mask",
       "their_mask_question": "What mask are THEY wearing?",
       "their_mask_note": "If they’re eating or drinking, select “no mask”",
-      "filtered_ensure_airflow": "Ensure sufficient flow through the filter for your room. For help on how to do this, see <1>here</1>.",
+      "filtered_ensure_airflow": "Ensure sufficient airflow through the filter for your room, <1>like so</1>.",
       "volume_header": "Volume",
       "volume_question": "How much is everyone talking?"
     },
-    "risk_note_about_household_members": "When estimating your risk of infection from a household member or partner/spouse we assume these interactions are indoors, unmasked, and undistanced.",
+    "risk_note_about_household_members": "Your estimated risk of infection from a household member or partner/spouse assumes these interactions are indoors, unmasked, and undistanced.",
     "explanationcard": {
       "person": "person",
       "person_plural": "people",
       "multiple_suffix": "x your weekly risk budget",
       "percentage_suffix": "% of your weekly risk budget",
-      "details_header_closed": "Show calculation details",
-      "details_header_open": "Hide calculation details",
+      "details_header_closed": "Show details",
+      "details_header_open": "Hide details",
       "details_overview_header": "Calculation",
       "details_person_risk": "Person Risk",
       "details_activity_risk": "Activity Risk",
       "details_steps_header": "Calculation Steps",
       "details_person_risk_number": "{{ person_risk }}-in-a-million chance",
-      "details_person_risk_explanation": "First, we calculate that each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID.",
-      "details_person_risk_explanation_housemate": "First, we calculate that each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID, not including their risk from living with you.",
+      "details_person_risk_explanation": "First, the calculation assumes each other person in this area has a <strong>{{ person_risk }}-in-a-million chance</strong> of currently having COVID.",
```

Comment:
It seems to me that you prefer to remove the voice and perspective of the authors ("we"), and adopt a neutral-sounding abstract perspective, is that right?

I think in general we don't prefer this style, and prefer to make it clear that we are humans making a judgment call, not abstract authorities.

But I'd love to hear what you think we might be missing in that.

Reply:
That is already clear (now) in the disclaimer.
Show me one program that re-iterates itself as a human entity at every turn. For good reason, that isn't the way to do it. It is neither practical nor sentient software.